modules/local/seqera_runs_dump/nextflow.config

```diff
@@ -1,11 +1,11 @@
 process {
     withName: 'SEQERA_RUNS_DUMP' {
         ext.args = { params.seqera_cli_extra_args ? params.seqera_cli_extra_args.split("\\s(?=--)") : '' }
-        ext.args2 = { params.skip_run_gantt ? '' : '--add-task-logs --add-fusion-logs' }
+        ext.args2 = { params.skip_run_gantt ? '' : '--add-task-logs' }
```

Comment:
Can't remember why we added `--add-task-logs` here when using `params.skip_run_gantt`. Do we need to move this setting into the process too?

Reply:
No, don't think so, we'd have to pass `params.skip_run_gantt` into the process, which doesn't seem sensible. 

The other option was switched to output the fusion logs depending on fusions status, which is native to the task, which this is not.